frontend/integration-tests/tests/secrets.scenario.ts

```diff
@@ -1,64 +1,119 @@
-import { $, element, browser, by, ExpectedConditions as until } from 'protractor';
+/*eslint max-nested-callbacks: ["error", 5]*/
+import { $, element, browser, by, ExpectedConditions as until, Key } from 'protractor';
 
 import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
 import * as crudView from '../views/crud.view';
 import * as secretsView from '../views/secrets.view';
 
-const SECRET_NAME = `secret-${testName}`;
-
-describe('Interacting with the create secret forms', () => {
+const SECRET_NAMES = {
+  webhook: 'webhook-secret',
+  basicSource: 'basic-source-secret',
+  sshSource: 'ssh-source-secret',
+};
```

Comment:
I would probably just use

```js
const webhookSecretName = 'webhook-secret';
```

inside `describe` for the test that uses it. Then each test should delete the secret it creates instead of doing an `afterAll`